### ISSUE TITLE
Baselines

### ISSUE THREAD

**Issue by marlens123** [RolnickLab/snowgalileo#11]
- [x] Data pre-processing for sklearn models: Dataset class and mask filling
- [x] Sklearn model implementations: RF, SVR, MLPRegressor 
- [x] Hyperparameter tuning ANN, SVR, RF (consider https://www.sciencedirect.com/science/article/pii/S003442572100328X#s0010)
- [ ] Check number of neurons in hidden layer _(on hold until test sets are available)_
- [ ] Add / remove normalization for sklearn models _(on hold until test sets are available)_
- [ ] Clean-up wandB logging _(on hold until test sets are available)_
- [ ] make training more efficient (e.g., through n_jobs) _(on hold until test sets are available)_
- [ ] GlobalSnowpack
- [x] handle remaining NaN values for SVR and MLP

**Comment by marlens123** [comment]
**Update 12/12/2025:**
- test for add / remove normalization waiting for evaluation data
- clean-up wandB logging waiting for evaluation data
- test for efficient training waiting for evaluation data
- in progress: GlobalSnowpack. Waiting for evaluation data to identify how to match GlobalSnowpack with evaluation bounding boxes. TODO: create script to access GlobalSnowpack with STAC
